src/Widgets/DataLabelWidget/DataLabelWidgetComponent.tsx

```diff
@@ -60,7 +61,11 @@ const AttributeValue = connect(function AttributeValue({
   if (showAs === 'currency') return <Currency value={attributeValue} />
   if (showAs === 'datetime') return <Datetime value={attributeValue} />
 
-  return <Text value={attributeValue} />
+  if (!dataItem) return <Text value={attributeValue} />
```

Comment:
Let's move the `!dataItem` check to the top of the function and return `'N/A'` right away.